src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractValidationServiceImpl.java

```diff
@@ -86,5 +102,55 @@ public boolean validateDate(Date date) {
         }
         return valid;
     }
+    
+    @Override
+    public boolean validateConcurStandardAccountingExtractDetailLine(ConcurStandardAccountingExtractDetailLine line) {
+        boolean valid = validateReportId(line.getReportId());
+        valid = validateAccouningLine(line) && valid;
+        return valid;
+    }
+    
+    private boolean validateReportId(String reportId) {
+        boolean valid = StringUtils.isNotEmpty(reportId);
+        if (valid) {
+            LOG.debug("validateReportId, found a valid report ID: " + reportId);
+        } else {
+            LOG.error("validateReportId, no valid report ID");
+        }
+        return valid;
+    }
+    
+    /*
+    private boolean validateAccountingInformation(ConcurStandardAccountingExtractDetailLine line) {
+        return validateRequiredElementsExists(line) && validateAccouningLine(line);
+    }*/
+
+    private boolean validateAccouningLine(ConcurStandardAccountingExtractDetailLine line) {
+        ConcurAccountInfo concurAccountInfo = new ConcurAccountInfo(line.getChartOfAccountsCode(), line.getAccountNumber(), 
+                line.getSubAccountNumber(), line.getJournalAccountCode(), line.getSubObjectCode(), line.getProjectCode(), line.getOrgRefId());
+        ValidationResult validationResults = getConcurAccountValidationService().validateConcurAccountInfo(concurAccountInfo);
```

Comment:
Does the ConcurAccountValidationService validate the object code? If so, the service may need to be updated to allow for the "Pending Client" object code, or the the replacement value for "Pending Client" will need to be passed to the validation service instead. (For the SAE-to-Collector processing, I still need to process rows which use that code, so I can't use this validation code for the Collector processing unless it can handle the "Pending Client" code.)